File: Python/3.Application in Industry Class/6.Eksplorasi dan Analisis Data COVID-19 Indonesia using Python/4.Status Code.py
import requests
resp = requests.get('https://storage.googleapis.com/dqlab-dataset/update.json', verify=False)
print(resp)

# Tugas
# Ada beberapa jenis status code yang umumnya dijumpai, antara lain:

# 200.Artinya permintaan sukses dipenuhi.
# 404.Artinya berkas yang diminta tidak dapat ditemukan.
# 500.Artinya akses permintaan ditolak.
# 501.Artinya terjadi kesalahan pada server.
# Informasi lengkap mengenai jenis-jenis status code dapat Anda pelajari di https://restfulapi.net/http-status-codes/

# Anda dapat mencetak resp secara langsung yang telah memberikan output status atas permintaan Anda melalui API. Sekarang coba lihatlah status atas permintaan yang telah Anda buat! Apakah permintaan Anda berhasil?

File: R/3.Application in Industry Class/5.Analisis Data COVID19 di Indonesia/4.Status Code.R
library(httr)
set_config(config(ssl_verifypeer = 0L))
resp <- GET("https://storage.googleapis.com/dqlab-dataset/update.json")
status_code (resp)

# Teori
# Ada beberapa jenis status code yang umumnya dijumpai, antara lain:

# 200 Artinya permintaan sukses dipenuhi.
# 404 Artinya berkas yang diminta tidak dapat ditemukan.
# 403 Artinya akses permintaan ditolak.
# 500 Artinya terjadi kesalahan pada server.
# Informasi lengkap mengenai jenis-jenis status code dapat diakses melalui :

# https://restfulapi.net/http-status-codes/

# Anda dapat menggunakan fungsi status_code() untuk mengetahui status atas permintaan Anda melalui API.